pkg/apps/generated/clientset/typed/apps/v1/deploymentconfig_expansion.go

```diff
@@ -0,0 +1,33 @@
+package v1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	kapi "k8s.io/kubernetes/pkg/api"
+	v1beta1 "k8s.io/kubernetes/pkg/apis/extensions/v1beta1"
+)
+
+type DeploymentConfigExpansion interface {
+	UpdateScale(deploymentConfigName string, scale *v1beta1.Scale) (result *v1beta1.Scale, err error)
+}
+
+var scaleCodec = kapi.Codecs.LegacyCodec(v1beta1.SchemeGroupVersion)
+
+// UpdateScale takes the top resource name and the representation of a scale and updates it. Returns the server's representation of the scale, and an error, if there is any.
+func (c *deploymentConfigs) UpdateScale(deploymentConfigName string, scale *v1beta1.Scale) (result *v1beta1.Scale, err error) {
+	// FIXME: make non-homogenous subresource GV client generation work
```

Comment:
I'll sync with @sttts tomorrow and try to fix it in client generation. Unless you do it first.